### PR TITLE
Tms dev

### PR DESCRIPTION
**Pull request by tmikeschu**
The ...0.csv file has updated headers now, and should be good to go for CSV.parsing.

### CODE REVIEW

**Review by AliSchlereth — COMMENTED** [review]
@tmikeschu We should delete lines 33-44 as well.  Will you go back in your code, delete, and re-push? Thanks!

**Comment by tmikeschu** [comment]
Thanks for this reminder! Considering we can always look back on the pdf for the table notes, I removed everything after the Wisconsin row.

**Comment by tmikeschu** [comment]
There is also a png screenshot of the SQL schema.

**Comment by AliSchlereth** [comment]
Thanks @tmikeschu ! Looks good. 

cc: @bermannoah

File: edu_data_0.csv [hunk 1]
```diff
@@ -1,13 +1,4 @@
...
-Table,4.—independent,charter,school,districts,by,district,type,and,state:,Fiscal,year,2013
-Median,current,expenditures,per,pupil,Districts,type,Students67
-Instruction,and,Number,in,reporting
-Total,instruction-related2,Support,services3,Total,Number,reporting5,Total,districts
-8,9,8,9,8,9,8,9,8,9,8,9,8,9
-State,Charter,Noncharter,Charter,Noncharter,Charter,Noncharter,Charter,Noncharter,Charter,Noncharter,Charter,Noncharter,Charter,Noncharter
-10
-med_spend_total_c, med_spend_total_nc, med_spend_instruction_related_c, med_spend_instruction_related_nc, med_spend_support_services_c, med_spend_support_services_nc, dist_total_c, dist_total_nc, dist_num_reporting_c, dist_num_reporting_nc, stud_total_c, stud_total_nc, stud_rep_dist_c, stud_rep_dist_nc
-Reporting,states,$8866,$10327,$5033,$6470,$3564,$3385,2720,10422,2291,9659,1210260,28198067,1108095,28159828
+state, med_spend_total_c, med_spend_total_nc, med_spend_instruction_related_c, med_spend_instruction_related_nc, med_spend_support_services_c, med_spend_support_services_nc, dist_total_c, dist_total_nc, dist_num_reporting_c, dist_num_reporting_nc, stud_total_c, stud_total_nc, stud_rep_dist_c, stud_rep_dist_nc
```
Comment (bermannoah) [inline edu_data_0.csv:1]: so clean! 👍

**Comment by bermannoah** [comment]
👍 👍 🎉

**Pull request merged**